docs/docs/zh/guide/task/flink.md

```diff
@@ -2,7 +2,11 @@
 
 ## 综述
 
-Flink 任务类型，用于执行 Flink 程序。对于 Flink 节点，worker 会通过使用 flink 命令 `flink run` 的方式提交任务。更多详情查看 [flink cli](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/deployment/cli/)。
+Flink 任务类型，用于执行 Flink 程序。对于 Flink 节点：
+
+(1) 当程序类型为 Java、Scala 或 Python 时，worker 使用 Flink 命令提交任务 `flink run`。更多详情查看 [flink cli](https://nightlies.apache.org/flink/flink-docs-release-1.14/docs/deployment/cli/) 。
+
+(2) 当程序类型为 SQL 时，worker 使用 Flink SQL 提交任务 `sql-client.sh`。更多详情查看 [flink sql client](https://nightlies.apache.org/flink/flink-docs-master/docs/dev/table/sqlclient/) 。
```

Comment:
Is it better to use ```worker 使用 `sql-client.sh` 来提交 Flink SQL 任务```? The same as en's doc.

Reply:
You're right, I'll fix it now